tests/integration/defs/test_e2e.py

```
@@ -28,8 +28,9 @@
 from defs.trt_test_alternative import (check_call, check_call_negative_test,
                                        check_output)
 
-from .common import (PluginOptions, convert_weights, prune_checkpoint,
-                     quantize_data, refit_model, venv_check_call)
+from .common import (PluginOptions, convert_weights, get_mmlu_accuracy,
```

Ruff (0.12.2)

Redefinition of unused `convert_weights` from line 27

(F811)